extension.js

```diff
@@ -159,9 +159,8 @@ function icon_should_be_visible(microphone_active) {
 }
 
 function show_osd(text, muted, level) {
-  const monitor = -1;
   const icon = Gio.Icon.new_for_string(get_icon_name(muted));
-  Main.osdWindowManager.show(monitor, icon, text, level);
+  Main.osdWindowManager.showAll(icon, text, level);
 }
 
```

Comment:
does this `.showAll()` also work on older gnome-shell versions? the [gnome-shell extension upgrade docs](https://gjs.guide/extensions/upgrading/gnome-shell-49.html#osdwindowmanager) suggest it's a new API, but it's not entirely clear to me.

if `.showAll()` is v49 only API, older versions should be dropped from the `metadata.json` file. (users on older versions will keep using the older version in that case, as i understand it)

Reply:
thanks. dropping support for older versions would've been fine for me as well (see previous comment), but this will do as well!